FlowCrypt/Functionality/Services/Account Server Services/EnterpriseServerApi.swift

```diff
@@ -152,7 +142,7 @@ class EnterpriseServerApi: EnterpriseServerApiType {
         )
 
         let response: MessageUploadResponse = try await performRequest(
-            email: sender,
+            email: details.from,
```

Comment:
This is a subtle thing: `from` and `current account` may not be the same thing in the future - once we add support for different `sendAs` aliases, in which case you may send an email from `alias@example.com` while on account `account@example.com`. So this would really need an `account` and `from` separately, or in the future maybe `idToken` and `from` separately. For now though, it's ok. I'll file an issue.